src/awkward/_connect/pyarrow/__init__.py

```diff
@@ -0,0 +1,124 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
```

Comment:
I like the fact that the code in this `__init__.py` is minimal; it's just about importing various parts of pyarrow.

Reply:
Thanks, this arrangement should allow for other code to do `from awkward._connect.pyarrow import ...` without too much anxiety, if the author wants.